manifests/repo.pp

```diff
@@ -188,7 +188,7 @@
         logoutput => on_failure,
       }
 
-      if $::mrepo_env {
+      if $mrepo_env {
```

Comment:
Not super clean?  ;)  If only.

The code currently does something like...
```puppet
$repo_command = "${mrepo_command} ${mrepo_options} ${name} ${mrepo_logging}"
exec { "Synchronize repo ${name}":
  command  => $repo_command,
  ...
```

So would actually need to default to `'> /var/log/mrepo.sync.log'` and not just a filepath?